catalog/spatial/country/spatial-country-local-impl/src/main/java/org/codice/ddf/spatial/country/converter/impl/LocalCountryCodeConverter.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.country.converter.impl;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nullable;
+
+import org.apache.commons.lang.StringUtils;
+import org.codice.ddf.spatial.country.converter.api.CountryCodeConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Multimap;
+
+import ddf.security.PropertiesLoader;
+
+public class LocalCountryCodeConverter implements CountryCodeConverter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LocalCountryCodeConverter.class);
+
+    private Multimap<String, String> countryCodesMap;
+
+    public List<String> convertFipsToIso3(@Nullable String fipsCountryCode) {
+        return convert(fipsCountryCode, true);
+    }
+
+    public List<String> convertIso3ToFips(@Nullable String iso3alphaCountryCode) {
+        return convert(iso3alphaCountryCode, false);
+    }
+
+    public String convertIso2ToIso3(String language, String alpha2CountryCode) {
+        Locale locale = new Locale(language, alpha2CountryCode);
```

Comment:
Since all we're doing here is delegate to Java's `Locale`, do we really need to make this part of the service? Especially given the fact that this code has nothing to do with the country mappings.